addons/sms/static/src/models/message_view.js

```diff
@@ -0,0 +1,57 @@
+/** @odoo-module **/
+
+import { patchRecordMethods } from '@mail/model/model_core';
+import { clear } from '@mail/model/model_field_command';
+// ensure that the model definition is loaded before the patch
+import '@mail/models/message_view';
+
+patchRecordMethods('MessageView', {
+    /**
+     * @override
+     */
+    _computeFailureNotificationIconClassName() {
+        if (!this.message) {
+            return clear();
+        }
+        if (this.message.message_type === 'sms') {
+            return 'fa fa-mobile';
+        }
+        return this._super();
+    },
```

Comment:
Since `clear()` would be the default behavior from _super, you could just do this ?

```javascript
 _computeFailureNotificationIconClassName() {
        if (this.message && this.message.message_type === 'sms') {
            return 'fa fa-mobile';
        }
        return this._super();
    },
```

Reply:
done